check_diff/tests/bash_commands.rs

```diff
@@ -0,0 +1,34 @@
+use check_diff::change_directory_to_path;
+use std::env;
+use std::ffi::OsStr;
+use tempfile::Builder;
+use tracing::error;
+
+#[test]
+fn cd_test() {
+    // create directory in current directory
+    let dir = Builder::new().tempdir_in("");
+
+    match dir {
+        Ok(d) => {
+            let dest_path = d.path();
+            let _ = change_directory_to_path(dest_path);
```

Comment:
Probably simpler to just `unwrap` in the test instead of silently ignoring the returned `Result`.

Reply:
the `Result` here is actually ignored because we are actually changing the `env` variable.